index.html

```diff
@@ -11853,9 +11853,9 @@ <h2>Definitions of States and Properties (all aria-* attributes)</h2>
 			<div class="state-description">
 				<p><a>Indicates</a> whether the <a>element</a> is exposed to an accessibility API. See related <sref>aria-disabled</sref>.</p>
```

Comment:
What if we changed this to:

<p>If set to true, hides an element and its entire subtree from assistive technology, regardless of whether it's visibly rendered.</p>

In particular:
* The current wording ("indicates whether the element is exposed to an accessibility API") sort of implies that both true and false could have meaning. Let's make it more clear that only aria-hidden=true means anything.
* I think it's confusing to say that aria-disabled is related. That seems like a stretch to me. I think it'd be more clear to leave that out.

Reply:
Indicates is one of the primary reserved terms (indicates, identifies, etc) that allude to the value type. How about this:
```suggestion
				<p><a>Indicates</a>, when set to true, that an <a>element</a> and its entire subtree are hidden from assistive technology, regardless of whether it's visibly rendered. See related <sref>aria-disabled</sref>.</p>
```

Reply:
Yep, I like that better. Thanks